src/rendering/src/services/SendService.tsx

```diff
@@ -0,0 +1,78 @@
+import { LayoutServiceContext } from '@sitecore-jss/sitecore-jss-nextjs';
+import Script from 'next/script';
+import { isEditingOrPreviewingPage } from '../helpers/LayoutServiceHelper';
+
+const SEND_WEBSITE_ID = process.env.NEXT_PUBLIC_SEND_WEBSITE_ID || '';
+export const isSendConfigured = !!SEND_WEBSITE_ID;
+let isSendInitialized = false;
+let cancelDelayedFunctions = false;
+
+declare global {
+  interface Window {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    mootrack: any;
+  }
+}
+
+export const SendScripts: JSX.Element | undefined = isSendConfigured ? (
+  <>
+    <Script id="sendTracker">{`
+      //load TrackerJS
+      !function(t,n,e,o,a){function d(t){var n=~~(Date.now()/3e5),o=document.createElement(e);o.async=!0,o.src=t+"?ts="+n;var a=document.getElementsByTagName(e)[0];a.parentNode.insertBefore(o,a)}t.MooTrackerObject=a,t[a]=t[a]||function(){return t[a].q?void t[a].q.push(arguments):void(t[a].q=[arguments])},window.attachEvent?window.attachEvent("onload",d.bind(this,o)):window.addEventListener("load",d.bind(this,o),!1)}(window,document,"script","//cdn.stat-track.com/statics/moosend-tracking.min.js","mootrack");
+      `}</Script>
+  </>
+) : undefined;
+
+export function initialize(context?: LayoutServiceContext): void {
```

Comment:
This will soon change to revert back to passing the pageState to this function. Marking the PR as waiting for changes for now until the Edge repo is fixed.